### PR TITLE
fix(grid styles): header position and background

### PR DESCRIPTION
**Pull request by SunsetTechuila**
before:

image

image

after:

image

image

### CODE REVIEW

**Review by theRealPadster — APPROVED** [review]
Ah nice, I think this is what I was going for originally. Thanks for catching the readme bug too 🙃

**Comment by theRealPadster** [comment]
> I don't think sticky header is a good idea tho

I personally found it really annoying to have to scroll all the way up to the top to change colour schemes or search. Maybe we can tighten the spacing up a bit so it's not as tall?

File: src/styles/components/_grid.scss [hunk 1]
```diff
@@ -6,16 +6,20 @@
   align-content: space-between;
   align-items: center;
   color: var(--spice-text);
+  background-color: var(--spice-main);
   display: flex;
   justify-content: space-between;
-  margin: 16px 0;
+  height: 64px;
```
Comment (theRealPadster) [inline src/styles/components/_grid.scss:12]:
```suggestion
  padding-bottom: 8px;
```

File: src/styles/components/_grid.scss [hunk 2]
```diff
@@ -6,16 +6,20 @@
...
 
   // To position the settings button + colour schemes
   position: sticky;
   flex-direction: row-reverse;
-  top: 80px;
+  top: 64px;
   z-index: 1;
```
Comment (theRealPadster) [inline src/styles/components/_grid.scss:18]:
```suggestion
  // Fix positioning on readme pages
  .contentSpacing:has(#marketplace-readme) & {
    top: 0;
  }
  z-index: 1;
```

File: src/styles/components/_grid.scss [hunk 3]
```diff
@@ -6,16 +6,20 @@
...
 }
+.contentSpacing:has(#marketplace-readme) .marketplace-header {
+  top: 0;
+}
```
Comment (theRealPadster) [inline src/styles/components/_grid.scss:22]:
```suggestion
  top: 64px;
  z-index: 1;
  
  // Fix positioning on readme pages
  .contentSpacing:has(#marketplace-readme) & {
    top: 0;
  }
}
```

**Comment by theRealPadster** [comment]
I would also remove this part. It doesn't let me suggest on unmodified lines I guess. 
```css
.marketplace-header__left {
  position: absolute;
  left: 0;
}
```

**Comment by theRealPadster** [comment]
Screenshot for reference
image

**Comment by SunsetTechuila** [comment]
> I would also remove this part.

Then the right and left elements will swap places

image

**Comment by SunsetTechuila** [comment]
> I would also remove this part. It doesn't let me suggest on unmodified lines I guess.

done

**Review by theRealPadster — APPROVED** [review]

**Comment by AliahX** [comment]
This seems to have caused another issue?
The marketplace header is now displayed on top of the page content even when scrolled all the way to the top of the page, obscuring headers and anything else at the top of the page.
Here is a screenshot showing the issue
Screenshot 2024-06-19 173048
image
This is a fix I made by messing around with the css a bit, but it affected non spicetify pages too.
Screenshot 2024-06-19 173210

**Comment by AliahX** [comment]
You're right yeah. idk why I sent this in the marketplace repo...

**Pull request merged**